romancal/datamodels/tests/test_datamodels.py

```diff
@@ -85,15 +84,15 @@ def _setup_list_of_l2_files(n, obj_type, tmp_path):
                 / f"test_model_container_input_as_list_of_filepaths_{i:02}.asdf"
             )
             # create an ASDF file with an L2 model
-            utils.mk_level2_image(filepath=filepath, shape=(100, 100))
+            rdm.WfiImageModel.make_default(shape=(8, 100, 100), filepath=filepath)
```

Comment:
Not sure it is meaningful, but the addition of the 8 here surprises me; I expect WfiImageModels to be 2 dimensionsal.

Reply:
This is something we should talk about. The issue is there are 3 dimensional fields such as `amp33` and `border_ref_pix_left`. So it needs a third dimension to properly flush those values out when guessing.

Unfortunately, the `make_default` function by its nature is not as "intelligent" about what shapes are what, so it requires that it has enough dimensions to fill out every array of the model.